rolling/distribution.yaml

```diff
@@ -3256,6 +3256,22 @@ repositories:
       url: https://github.com/mrpt-ros-pkg/pose_cov_ops.git
       version: master
     status: maintained
+  py_trees:
+    doc:
+      type: git
+      url: https://github.com/splintered-reality/py_trees.git
+      version: devel
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/stonier/py_trees-release.git
```

Comment:
@stonier FYI, our current best practice is to have the release repository for Rolling releases in https://github.com/ros2-gbp.  The main reason for that is so that we can automatically release this package into the next release (Iron) when the time comes.  See https://github.com/ros2-gbp/ros2-gbp-github-org/blob/latest/CONTRIBUTING.md on how to create one.

Reply:
Got it, thanks!